CODE_OF_CONDUCT.md

```
@@ -0,0 +1,69 @@
+**Git-Aura Code of Conduct**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)